src/main/java/org/occiware/tinom/aggregators/Median.java

```diff
@@ -0,0 +1,30 @@
+
+package org.occiware.tinom.aggregators;
+
+import java.util.Arrays;
+
+/**
+ * Median "mean" aggregation operator. Will take the middle of the vector of numbers.
+ *
+ * @author Mehdi Ahmed-Nacer
+ */
+public class Median implements Aggregation {
+
+    public static Median INSTANCE = new Median();
+
+    @Override
+    public double apply(double... values) {
+        if (values == null || values.length == 0)
```

Comment:
Your method has several return statements. Generally, it is better to only have a single one at the end. It makes debugging easier (only one location to put a break point). This is only a notice, I do not ask you to modify your code. Nor do I ask you to add unit tests...